backend/core/settings.py

```diff
@@ -360,7 +360,7 @@
 POST_LOGIN_REDIRECT_URL = os.environ.get("POST_LOGIN_REDIRECT_URL", f"{PUBLIC_URL}/dashboard")
 
 # whitelisting of users
-USE_WHITELIST = os.environ.get("USE_WHITELIST", False)
+USE_WHITELIST = eval(os.environ.get("USE_WHITELIST", False))

```

Comment:
`eval` cannot operate on boolean values so the default to get() needs to be a string as well. Also `eval` is probably not very safe, better use `ast.literal_eval()`